helper/helper_web.py

```diff
@@ -1,8 +1,9 @@
 from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
 from helper.helper_base import HelperFunc
  
 def get_browser(browser):
     if browser == "chrome":
         options = webdriver.ChromeOptions()
         options.add_argument('headless')
```

Comment:
Does this get used? I can see you were passing it to the function before but I can't see how it would get in now.
Is it that it doesn't, but the only option we were using was `headless` and now that's moot with `Remote()`? If so, these two lines just want deleting?

Reply:
removed those extraneous lines of code.